content/the-ember-times-issue-179.md

```diff
@@ -26,15 +28,25 @@ tags:
 
 ---
 
-## [2. Section title in sentence case 🐹](section-url)
+## Readers' Question: What’s the current status of the Embroider project? 🧵
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+<Amy's note: Am wondering if this would be better to publish when [https://github.com/thoov/rfcs/blob/enable-embroider/text/0000-enable-embroider.md](https://github.com/thoov/rfcs/blob/enable-embroider/text/0000-enable-embroider.md) is live on the emberjs/rfcs repo?>
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+What's the current status of the Embroider project? [Embroider](https://github.com/embroider-build/embroider) is nearing a 1.0 release. The official cutover point will be when Ember CLI starts generating new apps with Embroider by default. 🙌
+
+If you're not familiar with Embroider, Embroider is a new build pipeline that will provide a way to compile Ember apps into spec-compliant, modern JavaScript. Using Embroider will make it easier to take advantage of code splitting and tree shaking in your app.
+
+Several large, heavily-tested Ember apps are already shipping to production with Embroider! However, there are some risks if you choose to use Embroider in production today. (You're likely to discover that some Ember addons don't work or will break your build. Also, Embroider's own configuration options are subject to change, so you'll need to read the [CHANGELOG.md](https://github.com/embroider-build/embroider/blob/master/CHANGELOG.md) to check for any gotchas when updating Embroider packages in your app.)
+
+Embroider has been a long time coming, and the future is looking bright. Embroider won't be turned on in the next Ember CLI 3.27 release, but there will be an RFC coming soon.
```

Comment:
I think it'd be good to highlight the delay. If timing works out, let's provide a link to the RFC.

```suggestion
Embroider has been a long time coming, and the future is looking bright. **Embroider won't be turned on in the next Ember CLI 3.27 release**. There will be an RFC coming soon.
```

Reply:
Thanks!! 

Brought it up at the Learning Meeting today. Let's see if the RFC lands tomorrow (I just followed that repo), as could def write that "actionable" news Friday night. It can go to next issue if not! 

I can tweak this and add more background info from EmberConf in the meantime...

Reply:
https://github.com/emberjs/rfcs/pull/746 woot!